maven/src/org/netbeans/modules/maven/queries/RepositoryMavenCPProvider.java

```diff
@@ -179,7 +179,7 @@ private ClassPathImplementation createCompileCPI(MavenProject project, File bina
         items.add(ClassPathSupport.createResource(FileUtil.urlForArchiveOrDir(binary)));
         if (project != null) {
             for (Artifact s : project.getCompileArtifacts()) {
-                File file = s.getFile();
+                File file = s.getFile(); // TODO perhaps needs patching
```

Comment:
I guess you should make your mind. There is nobody to answer that TODO question otherwise. Or am I wrong @mkleint?

Reply:
As above, this is something that is probably wrong, but does not seem like a priority to fix.